lighthouse-core/gather/connections/cri.js

```diff
@@ -74,10 +75,16 @@ class CriConnection extends Connection {
         });
         response.on('end', _ => {
           if (response.statusCode === 200) {
-            resolve(JSON.parse(data));
-            return;
+            try {
+              resolve(JSON.parse(data));
+              return;
+            } catch (e) {
+              if (data === 'Target is closing') {
+                return resolve({message: data});
+              }
```

Comment:
Since you have the try/catch now, want to add a `return reject(e)` here, even though it wasn't caught in the promise chain before?